recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
 {% set name = "lfortran" %}
-{% set version = "0.36.1" %}
+{% set version = "0.37.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   - url: https://github.com/lfortran/lfortran/releases/download/v{{ version }}/{{ name }}-{{ version }}.tar.gz
-    sha256: 2ffed05a6da827a1e81312ce385927bc5da3d828f774986466d6609332a0228f
+    sha256: 1e31898a8c8e8116ce8968f836e68ead06c5a5b4998b4d6cf6ae26be8301db7d
```

Comment:
From where do we get this hash?

Reply:
I think you can do:

```console
lfortran % curl -sL https://github.com/lfortran/lfortran/releases/download/v0.37.0/lfortran-0.37.0.tar.gz | openssl sha256
SHA2-256(stdin)= 1e31898a8c8e8116ce8968f836e68ead06c5a5b4998b4d6cf6ae26be8301db7d
```

where you can install `openssl` on mac using:

```console
brew install openssl
```

Reply:
Okay, thank you Ubaid! :)

Reply:
You can also just run `conda build` on the wrong hash, and it will tell you the right hash.

Reply:
This is interesting 😆